content/en/docs/components/pipelines/sdk/v2/v2-component-io.md

```diff
@@ -113,7 +113,7 @@ def train(
 
     with open(model.path, 'w') as output_file:
         for i in range(num_steps):
-            output_file.write('Step {}\n{}\n=====\n'.format(i, line))
+            output_file.write(f"Step {i}\n=====\n")
```

Comment:
nit: use single quotes for consistency within the file.

Reply:
good catch. i've pushed a new commit that fixes this